docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
+Configuring the Connection Profile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. User initiates association to WLAN AP and transmits MAC address.  If user accesses network via a registered device in PacketFence go to 8
-2. The WLAN controller transmits MAC address via RADIUS to the PacketFence server to authenticate/authorize that MAC address on the AP
-3. PacketFence server conducts address audit in its database.  If it does not recognize the MAC address go to 4. If it does go to 8.
-4. PacketFence server directs WLAN controller via RADIUS (RFC2868 attributes) to put the device in an "unauthenticated role“ (set of ACLs that would limit/redirect the user to the PacketFence captive portal for registration, or we can also use a registration VLAN in which PacketFence does DNS blackholing and is the DHCP server)
-5. The user's device issues a DHCP/DNS request to PacketFence (which is a DHCP/DNS server on this VLAN or for this role) which sends the IP and DNS information.  At this point, ACLs are limiting/redirecting the user to the PacketFence's captive portal for authentication.  PacketFence fingerprints the device (user-agent attributes, DHCP information & MAC address patterns) to which it can take various actions including:  keep device on registration portal, direct to alternate captive portal, auto-register the device, auto-block the device, etc.    If the device remains on the registration portal the user registers by providing the information (username/password, cell phone number, etc.).  At this time PacketFence could also require the device to go through a posture assessment (using Nessus, OpenVAS, etc.)
-6. If authentication is required (username/password) through a login form, those credentials are validated via the Directory server (or any other authentication sources - like LDAP, SQL, RADIUS, SMS, Facebook, Google+, etc.) which provides user attributes to PacketFence which creates user+device policy profile in its database.
-7. PacketFence performs a Change of Authorization (RFC3576) on the controller and the user must be re-authenticated/reauthorized, so we go back to 1
-8. PacketFence server directs WLAN controller via RADIUS to put the device in an "authenticated role“, or in the "normal" VLAN
+Next we need to configure the connection profile in PacketFence. That is required so that PacketFence knows how to handle a connection coming from the wired network or WiFi network. In our case, we will modify the default connection profile to use our Microsoft Active Directory authentication source and also to let PacketFence know to automatically register any devices that successfully authenticate using 802.1X on the default connection profile.
+ 
+From 'Configuration->Policies and Access Control->Connection Profiles', click on the 'default' profile. Make sure 'Automatically register devices' is checked and make sure your newly created Active Directory authentication source is selected in 'Sources'. Click on 'Save'.
```

Comment:
IMO, we'd be better off creating a new profile for this

Reply:
How about we always create a new profile and never touch the default one?

I think it makes things easier for users, to grasp things and avoid the "!?!?# how did I land here?" - something that @lzammit ask himself from bed when he wakes up every single morning.

Reply:
I 100% agree with creating a new profile each time